static/templates/actors/npc/sheet.html

```diff
@@ -6,10 +6,10 @@
         {{> "systems/pf2e/templates/actors/npc/partials/header.html"}}
         <nav class="sheet-tabs tabs" data-group="primary">
             <a class="item" data-tab="main">{{localize "PF2E.NPC.MainTab"}}</a>
-            <a class="item" data-tab="inventory">{{localize "PF2E.NPC.InventoryTab"}}</a>
+            <a class="item {{#if (not document.inventory.size)}}disabled{{/if}}" data-tab="inventory">{{localize "PF2E.NPC.InventoryTab"}}</a>
             <a class="item {{#if (not isSpellcaster)}}disabled{{/if}}" data-tab="spells">{{localize "PF2E.NPC.SpellsTab"}}</a>
-            <a class="item" data-tab="effects">{{localize "PF2E.NPC.EffectsTab"}}</a>
-            <a class="item" data-tab="notes">{{localize "PF2E.NPC.NotesTab"}}</a>
+            <a class="item {{#if (not hasConditionOrEffect)}}disabled{{/if}}" data-tab="effects">{{localize "PF2E.NPC.EffectsTab"}}</a>
+            <a class="item {{#with (not data.details.publicNotes data.details.privateNotes)}}disabled{{/with}}" data-tab="notes">{{localize "PF2E.NPC.NotesTab"}}</a>
```

Comment:
use unless and or instead. `with` is superfluous to if/unless if all you need is evaluation. not only applies to a single argument, ignoring the second one.

Reply:
Updated with to unless and replaced not with or to get required logic to check if either field has data. Thanks again.